lib/vagrant/util/experimental.rb

```diff
@@ -0,0 +1,48 @@
+module Vagrant
+  module Util
+    class Experimental
+      VALID_FEATURES = [].freeze
```

Comment:
I don't think we want to have a hard restriction on allowed names so I would vote to remove this.